frontend/components/MainPage/header/AgentButton/AgentNotRunningButton.tsx

```diff
@@ -32,10 +33,57 @@ import { AgentConfig } from '@/types/Agent';
 import { delayInSeconds } from '@/utils/delay';
 import { updateServiceIfNeeded } from '@/utils/service';
 
+/**
+ * Health check hook
+ */
+const useHealthCheck = () => {
+  const { selectedAgentType } = useServices();
+  const { goto: gotoPage } = usePageState();
+
+  return useCallback(
+    async (serviceId?: string) => {
+      if (!serviceId) return true; // TODO: check
+
+      try {
+        const response = await ServicesService.getDeployment(serviceId);
+        const healthcheck = response.healthcheck;
+        if (isEmpty(healthcheck) || !healthcheck.env_var_status?.needs_update) {
+          return true;
+        }
+
+        if (selectedAgentType !== AgentType.Memeooorr) {
+          return true;
+        }
+
+        const reason =
+          healthcheck.env_var_status?.env_vars?.['TWIKIT_COOKIES']?.reason;
+        const isXSuspended = reason?.includes('suspended');
```

Comment:
I'd say we rather display either all or the first one of the messages regardless the env name or what text it contains. you can check the code here for some more context https://github.com/dvilelaf/meme-ooorr/blob/eda7ac943e55de20c36694cb5d4586b0ab889fb6/packages/dvilela/skills/memeooorr_abci/behaviour_classes/base.py#L171